src/content/docs/licenses/open-source/oss-category.mdx

```diff
@@ -0,0 +1,121 @@
+redirects:
+  - https://opensource.newrelic.com/oss-category
+freshnessValidatedDate: never
+---
+
+Every public repository in the New Relic and New Relic Experimental GitHub organizations leverages one of the following open source categories:
+
+<CollapserGroup>
+  <Collapser
+    id="community-project"
+    title="Community Project"
+  >
+    This code is developed in the open with input from the community through issues and PRs. There is an active maintainer team within New Relic, troubleshooting support in the New Relic Explorers Hub, and the documentation is available in the project repository.
+    <Callout variant="important">
+    This is an official New Relic open source community project, supported by maintainers within New Relic.
+    </Callout>
+
+    **Requirements**
+    - Complies with New Relic's legal and security requirements for open source software
+    - Contains the appropriate New Relic open source category header in the repository's README file
+    - Has active maintainer / maintainers, including at least one Relic
+    - Troubleshooting support via the New Relic Explorers Hub
+    - Issues and PR’s managed in GitHub
+    - Documentation reviewed by the New Relic documentation team
+    - Linted code
+    - An automated release pipeline
+  </Collapser>
+
+  <Collapser
+    id="community-plus"
+    title="Community Plus"
+  >
+    This code is developed in the open with input from the community through issues and PRs. A New Relic engineering team serves as the maintainer. Troubleshooting support is available in the New Relic Explorers Hub, and documentation is available in the project repository and [New Relic Docs](https://docs.newrelic.com).
+
+    <Callout variant="important">
+    This code is currently maintained by New Relic engineering teams and delivered here in GitHub. See the README for troubleshooting and defect reporting instructions.
+    </Callout>
+
+    **Requirements**
+    - Complies with New Relic's legal and security requirements for open source software
+    - Contains the appropriate New Relic open source category header in the repository's README file
+    - Is maintained by a New Relic engineering team
+    - Troubleshooting support via the New Relic Explorers Hub
+    - Issues and PR’s managed in GitHub
+    - For additional details on support options, see the [Open Source Support Policy](/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings?_gl=1*1ftd2ji*_gcl_aw*R0NMLjE3MjE3NDA0NzIuQ2p3S0NBandxZjIwQmhCd0Vpd0F0N2R0ZGQxa01LaHBmazl1OFNQZTlTRU02eHZ1UThaZXk3dVdoQ3F2Y09iX2JzOFJpbENRUDlwZVpob0NGdkFRQXZEX0J3RQ..*_gcl_au*MTUzNzk1ODMxOS4xNzIwNDI5NDM0*_ga*NzE1OTA0NDA1LjE3MjA0Mjk0MzU.*_ga_R5EF3MCG7B*MTcyNzQxNzgxOS4yMjIuMS4xNzI3NDE4OTQxLjYwLjEuMjI2NDAzOTc2)
+    - Documentation reviewed by the New Relic documentation team
+    - Linted code
+    - An automated release pipeline
+  </Collapser>
+
+  <Collapser
+    id="one-catalog"
+    title="New Relic One Catalog"
```

Comment:
We don't use the brand "New Relic One" anymore. It should just be "New Relic catalog"

Reply:
Got it